backend/src/plugins/Mutes/events/ReapplyActiveMuteOnJoinEvt.ts

```diff
@@ -25,7 +26,7 @@ export const ReapplyActiveMuteOnJoinEvt = mutesEvt({
       if (!member.isCommunicationDisabled()) {
         const expiresAt = mute.expires_at ? moment.utc(mute.expires_at).valueOf() : null;
         const timeoutExpiresAt = getTimeoutExpiryTime(expiresAt);
-        await member.disableCommunicationUntil(timeoutExpiresAt);
+        await member.disableCommunicationUntil(timeoutExpiresAt).catch(noop);
```

Comment:
Shouldn't we send bot alerts instead of ignoring the error (in the other files too)?